go.mod

```diff
@@ -1,5 +1,11 @@
 module github.com/stretchr/objx
 
-go 1.13
+go 1.20
```

Comment:
Is the version bump really nessary? What in the impact on users running go.19 or lower?

Reply:
We definitely should not change this to 1.20. It’d mean the new version won’t work with any go < 1.20